src/dao/providers/strava.ts

```diff
@@ -45,6 +69,21 @@ export default class StravaProviderDAO extends WithLog
       avatarUrl: rawUser.profile,
     };
   }
+
+  private convertActivity(rawActivity: RawStravaActivity): StravaActivity {
+    return {
+      userId: this.userId,
+      name: rawActivity.name,
+      type: rawActivity.type,
+      startDate: new Date(rawActivity.start_date),
+      movingTime: rawActivity.moving_time,
+      totalTime: rawActivity.elapsed_time,
+      distance: rawActivity.distance,
+      elevation: rawActivity.total_elevation_gain,
+      provider: 'strava',
+      providerId: '' + rawActivity.id,
```

Comment:
Interesting cast to string. It's a nit, but `rawActivity.id.toString()` might be more clear.

Reply:
I don't know why I went that route. 🤕